tasks/main.yml

```diff
@@ -48,6 +49,7 @@
     dest: "/usr/local/sbin/{{ item }}"
     mode: "0755"
   with_items:
+    - cert-watcher
```

Comment:
This isn't deployed to production (or dev or stage) yet.  I first wanted to get a review.